etc/notebooks/test/test_plotting_libs.ipynb

```diff
@@ -385,7 +386,7 @@
    },
    "outputs": [],
    "source": [
-    "show(p)"
+    "handle = show(p, notebook_handle=True)"
```

Comment:
Hmm, I only see that error with Bokeh 0.11.1. Did you update to Bokeh 0.12.3 in your environment?